AdminPortal/src/data/data.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+      "id": 1,
+      "dateApplied": "2022-01-01",
+      "name": "John Doe",
+      "mobileNumber": "+1 123 456 7890",
+      "licenseID": "A1234567",
+      "licensePlate": "ABC-123",
+      "isChecked": false
```

Comment:
I'd prefer we don't make a separate folder and file just for dummy data, can we add this under pages or inside the dashboard file for now?

Reply:
makes sense, have changed that